js/index.js

```diff
@@ -39,23 +39,62 @@ function enterPressed (gameState) {
         resultMessage(result)
     }
 }
-
+console.log(gameState.expectedWord)
 function checkWord(gameState) {
     let correctLetters = 0
     let correctPositions = 0
-    for (let i = 0; i < gameState.attemptedWord.length; i++) {
-        if (gameState.attemptedWord[i] === gameState.expectedWord[i]) {
+    gameState.expectedWord = ['E','N','T','E','R']
+    // console.log(gameState.expectedWord)
+    let expectedWordCopy = gameState.expectedWord.slice()
+console.log(expectedWordCopy, 'pre any loop')
+    for (let i = 0; i < 5; i++) {
+        const tile = document.querySelector(`.row${gameState.attemptCounter}` + ' .tile' + i)
+        // const onscreenTile = document.querySelector('.key' + gameState.attemptedWord[i])
+        if (gameState.attemptedWord[i] === expectedWordCopy[i]) {
             correctPositions++
+            expectedWordCopy[i] = ' '
+            const onscreenKey = document.querySelector('.' + gameState.attemptedWord[i].toLowerCase())
+            gameState.attemptedWord[i] = '0'
+            tile.classList.add('correct-position')
+            onscreenKey.classList.add('correct-position')
```

Comment:
Because you re-use these classes, create a const to contain 'correct-position' - I would stick it at the top of checkWord

Reply:
done